plugins/woocommerce/includes/gateways/paypal/class-wc-gateway-paypal.php

```diff
@@ -640,6 +650,80 @@ public function admin_scripts() {
 		wp_enqueue_script( 'woocommerce_paypal_admin', WC()->plugin_url() . '/includes/gateways/paypal/assets/js/paypal-admin' . $suffix . '.js', array(), $version, true );
 	}
 
+	/**
+	 * Enqueue scripts.
+	 */
+	public function enqueue_scripts() {
+		if ( 'no' === $this->enabled ) {
+			return;
+		}
+
+		$version   = Constants::get_constant( 'WC_VERSION' );
+		$client_id = get_option( 'paypal_client_id' );
+
+		if ( ! $client_id || ! is_checkout() ) {
+			return;
+		}
+
+		// Load PayPal JS SDK dynamically with params.
+		$query = [
+			'client-id'       => $client_id,
+			'intent'          => $this->intent,
+			'buyer-country'   => 'US',
+			'locale'          => get_locale(),
+			'merchant-id'     => $this->email,
+			'components'      => 'buttons,funding-eligibility,messages',
+			'disable-funding' => 'card,applepay',
+			'enable-funding'  => 'venmo,paylater',
+			'currency'        => get_woocommerce_currency(),
+		];
+
+		$sdk_host = $this->testmode ? 'https://www.sandbox.paypal.com/sdk/js' : 'https://www.paypal.com/sdk/js';
+		
+		// Add PayPal JS SDK script.
+		wp_register_script( 'paypal-sdk', add_query_arg( $query, $sdk_host ), [], null, false );
```

Comment:
Should we choose a more specific name, e.g. `paypal-standard-sdk`. Thinking of how likely we'll have a script name collision in the event that the PPCP extension is also installed.